hooks/useHydration.jsx

```diff
@@ -0,0 +1,18 @@
+import useStore from './useStore';
+import { useState, useEffect } from 'react';
+
```

Comment:
this hook is a little 'hack' to bypass a current problem involving zustand and SSR.